Structure_Engine/Create/MaterialFragments/Aluminium.cs

```diff
@@ -0,0 +1,65 @@
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.ComponentModel;
+using BH.oM.Reflection.Attributes;
+using BH.oM.Physical.Materials;
+using BH.oM.Structure.MaterialFragments;
+using BH.oM.Geometry;
+
+namespace BH.Engine.Structure
+{
+    public static partial class Create
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        [Description("Creates a structural Aluminium material fragment to be used on analytical structural elements, or as a fragment of the physical material.")]
+        [InputFromProperty("youngsModulus")]
+        [InputFromProperty("poissonsRatio")]
+        [InputFromProperty("thermalExpansionCoeff")]
+        [InputFromProperty("density")]
+        [InputFromProperty("dampingRatio")]
+        [InputFromProperty("embodiedCarbon")]
+        [Output("aluminium", "The created tructural Aluminium material fragment.")]
+        public static Aluminium Aluminium(string name, double youngsModulus = 70000000000, double poissonsRatio = 0.34, double thermalExpansionCoeff = 0.000023, double density = 2710, double dampingRatio = 0, double embodiedCarbon = 7.9)
+        {
+            return new Aluminium()
+            {
+                Name = name,
+                Density = density,
+                YoungsModulus = youngsModulus,
+                PoissonsRatio = poissonsRatio,
+                ThermalExpansionCoeff = thermalExpansionCoeff,
+                DampingRatio = dampingRatio,
+                EmbodiedCarbon = embodiedCarbon,
+            };
+        }
```

Comment:
Why are there no default values in the oM which would make this redundant

Reply:
Get your point here, and in general agree, but, it felt wrong to have default values in the oM, but less wrong in the create method. Sort of, defaulting a method feels less set in stone than defaulting a full object. Not sure how solid that argument is, but that is why.
Also, opted to keep these from ongoing discussion in https://github.com/BHoM/BHoM_Engine/pull/1474 , where we might want to switch where default values for things like embodied carbon is being pulled from.